server/graphql/types/src/types/vaccine_course_dose.rs

```diff
@@ -0,0 +1,43 @@
+use async_graphql::*;
+
+use repository::vaccine_course::vaccine_course_dose_row::VaccineCourseDoseRow;
+
+#[derive(PartialEq, Debug)]
+pub struct VaccineCourseDoseNode {
+    pub vaccine_course_dose: VaccineCourseDoseRow,
+}
+
+#[Object]
+impl VaccineCourseDoseNode {
+    pub async fn id(&self) -> &str {
+        &self.row().id
+    }
+
+    pub async fn dose_number(&self) -> &i32 {
+        &self.row().dose_number
+    }
```

Comment:
Did you want to remove the dose number in this PR or make a follow-up issue?

Reply:
Will do here, thanks Carl 🙏